packages/logger/README.md

```diff
@@ -1,4 +1,320 @@
+});
+app.sequence(MySequence);
+
+const logger = app.getSync('logger');
+logger.info('My application has started.');
+
+```
+Below is an example implementation of sequence.  We've studied @inject as a parameter decorator.  The four `sequence.actions` keys are bound to the corresponding essential sequence actions.  Since each of the essential sequence actions are implemented by LoopBack Next and in most cases, you can simply inject them in your custom sequence.
+
+The custom sequence below is defined to do something special with req and res objects.  That's the sole purpose of MySequence.
+```
+import {DefaultSequence, FindRoute, InvokeMethod, Reject, Send, inject} from '@loopback/core';
+
+class MySequence extends DefaultSequence {
+  constructor(
+    @inject('sequence.actions.findRoute') protected findRoute: FindRoute,
+    @inject('sequence.actions.invokeMethod') protected invoke: InvokeMethod,
+    @inject('sequence.actions.send') public send: Send,
+    @inject('sequence.actions.reject') public reject: Reject,
+  ) {
+    super(findRoute, invoke, send, reject);
+  }
+
+  async handle(req: ParsedRequest, res: http.ServerResponse) {
+
+    ... we can grab req and res and do something here.  This is the purpose
+    ... of implementing MySequence.
+
+    await super.handle(req, res);
+  }
+}
+
+```
+
+
+# Bonus
+
+### How to test/validate API Spec
+
+As we saw in early part of this blog, the API Spec is attached to the application controller.
+Once attached, the loaded API spec can be accessed by `app.getApiSpec()`
+
+LoopBack Next provides the OpewnAPI spec validator, validateApiSpec as part of the lestlab.
+
+```
+const validateApiSpec = require('@loopback/testlab').validateApiSpec;
+
+describe('validate Api Spec', () => {
+  let app = createApp(); // instantiates the app, its controllers, and the API spec.
+  let apiSpec = app.getApiSpec();
+
+  it('apiSpec is valid', async () => {
+    await validateApiSpec(apiSpec);
+  });
+});
+
+```
+
+### How to debugt API Spec
+
+[The Swagger Editor](https://editor.swagger.io) is useful to interactively debug your OpenAPI specs.
+You can start with JSON or YAML.  Just cut and paste the helloworld API spec (below) into the swagger editor window.  It will ask you to use JSON or covert it to YAML and build the API spec in YAML.
+```
+{
+  "swagger": "2.0",
+  "basePath": "/",
+  "info": {
+    "title": "LoopBack Application",
+    "version": "1.0.0",
+  },
+  "paths": {
+    "/helloworld": {
+      "get": {
+        "x-operation-name": "helloWorld",
+        "parameters": [
+          {
+            "name": "name",
+            "in": "query",
+            "description": "Your name.",
+            "required": false,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Returns a hello world with your (optional) name."
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+### LoopBack Next globalization, heh?
```

Comment:
Ha, globalization is a good point that we have not considered yet. I think we should be extracting strings from the original TypeScript sources, what do you think, @Setogit?

Let's discuss globalization here: https://github.com/strongloop/loopback-next/issues/493

Important: globalization is out of scope of the initial beta release, but I think it should be a must-have for GA.